app/views/questionnaire.py

```diff
@@ -170,13 +176,14 @@ def post_household_composition(routing_path, **kwargs):
     return _render_page(routing_path, block, current_location, post_form=form)
 
 
-@questionnaire_blueprint.route('thank-you', methods=['GET'])
-def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
-    survey_completed_metadata = current_app.eq['session_storage'].get_survey_completed_metadata()
-    schema = load_schema_from_params(eq_id, form_type)
+@post_submission_blueprint.route('/thank-you', methods=['GET'])
+def get_thank_you():
+    session_data = current_app.eq['session_storage'].get_session_data()
+    metadata = get_metadata(current_user)
 
-    if survey_completed_metadata:
-        metadata_context = build_metadata_context_for_survey_completed(survey_completed_metadata)
+    if not metadata:
+        schema = load_schema_from_params(session_data.eq_id, session_data.form_type)
```

Comment:
If the version was updated while the user was completing the questionnaire, when they reach the thank-you page it will error.
We should handle this somehow. maybe populate the session_data on submit if it isn't present

Reply:
I'll remove the created_at from the debug message as not really needed. I'll need to create an update for session_data that can be checked at submit.